common/artifacts/000_wc_artifacts.txt

```diff
@@ -332,20 +345,55 @@ atiesh = {
 	stacking = no
 	indestructible = yes
 
-	learning = 3
+	learning = 2
 	health = 0.5
 	combat_rating = 20
 
-	monthly_character_prestige = 2
+	monthly_character_prestige = 1.5
 
+	vassal_opinion = 5
+	
 	text_effect_powerful_magical_artifact = 1
 
 	picture = "GFX_inv_staff_medivh"
 	slot = weapon
 }
 
-# Rare
+# Epic
+hammer_of_the_lightbringer = {
+	quality = 5
+	flags = { combat_weapon hammer }
+
+	active = {
+		is_adult = yes
+		has_class_trait_trigger = yes
+		OR = {
+			is_paladin_class_trigger = yes
+			is_warrior_class_trigger = yes
+		}
```

Comment:
There is no meaning in checking any class trait if you then check specific class

Reply:
Ok. I don't know why I put it there. probably took that from some other requirement from earlier and since it worked I didn't delete it anymore. I'll delete line 369 then

Reply:
fixed